src/com/aca/Management/Manager.java

```diff
@@ -0,0 +1,138 @@
+package com.aca.Management;
+
+import com.aca.Functionaluty.AudioRecorders;
+import com.aca.Functionaluty.Gadget;
+import com.aca.Functionaluty.VideoRecorders;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static java.lang.String.valueOf;
+
+
+public class Manager {
+    private List<Gadget> gadgets = new ArrayList<>();
+    private int soldAmount = 0;
+    List<Gadget> nonWorkingGadgets = new ArrayList<>();
+
+    public Manager(List<Gadget> gadgets)
+    {
+        this.gadgets = gadgets;
+    }
+
+    public void sellItem (String name)  {
+        Logger logger = Logger.getGlobal();
+        for(Gadget gadget: gadgets)
+        {
+            if(gadget.getName().equals(name))
+            {
+                if(gadget.getCount()>0) {
+                    gadget.setCount(gadget.getCount() - 1);
+                    soldAmount += gadget.getPrice();
+                    try {
+                        BufferedWriter tradedetails = new BufferedWriter(new FileWriter("src/com/aca/input/trading.csv"));
+                        tradedetails.append(gadget.getName());
+                        tradedetails.append(",");
+                        tradedetails.append(valueOf(gadget.getPrice()));
+                        tradedetails.newLine();
```

Comment:
instead of these line use toString, I want see how colors of pens also in trading.csv